src/mmw/js/src/modeling/tests.js

```diff
@@ -187,7 +195,15 @@ describe('Modeling', function() {
                 App.user.set('id', 1);
                 var project = getTestProject(),
                     view = new views.ProjectMenuView({ model: project }),
-                    projectResponse = '{"id":21,"user":{"id":1,"username":"test","email":"test@azavea.com"},"scenarios":[],"name":"Test Project","area_of_interest":{},"is_private":true,"model_package":"tr-55","created_at":"2015-06-03T20:09:11.988948Z","modified_at":"2015-06-03T20:09:11.988988Z"}';
+                    projectResponse = '{"id":21,"user":{"id":1,"username":"test","email":"test@azavea.com"},"scenarios":[],"name":"Test Project","area_of_interest":{},"is_private":true,"model_package":"tr-55","created_at":"2015-06-03T20:09:11.988948Z","modified_at":"2015-06-03T20:09:11.988988Z"}',
+                    postSaveMenuItems = [
+                        'Share',
+                        'Make Public', 'Make Private',
```

Comment:
We've set the privacy to private in the faked response so we know that the string should be "Make Public". Why allow the alternate case. If that showed up that would actually be an error.